packages/gatsby-plugin/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@prodo/gatsby-plugin",
```

Comment:
We should probably follow the [Gatsby plugin naming conventions](https://www.gatsbyjs.org/docs/naming-a-plugin/) and name this `gatsby-plugin-prodo`.

Reply:
yeah I wasn't sure whether to respect prodo's convention or gatsby's hahaha